cloudformation/ec2-asg.yaml

```diff
@@ -349,6 +365,19 @@ Resources:
       Protocol: HTTP
       UnhealthyThresholdCount: 5
       VpcId: !Ref 'VPC'
+  ALBMetricsTargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      HealthCheckIntervalSeconds: 30
+      HealthCheckTimeoutSeconds: 5
+      HealthyThresholdCount: 3
+      HealthCheckPath: /ping
```

Comment:
@maaquib Shouldn't this be a health check for the metric endpoint vs using the same /ping for all the endpoints?

Reply:
@chauhang We only have a single health check endpoint right now. I'll create an issue to have health checks for all the APIs

Reply:
@maaquib Created #735